### PR TITLE
[Manta] Configre manta testnet chain spec

### PR DESCRIPTION
**Pull request by Dengjianping**
This PR implements the chain spec for Manta testnet:
- [x] controller key 
- [x] stash key 
- [x] grandpa key 
- [x] babe key 
- [x] root key 
- [x] peer id 
- [x] initial token distribution

### CODE REVIEW

File: runtimes/manta/Cargo.toml [hunk 1]
```diff
@@ -20,9 +20,10 @@ substrate-build-script-utils = '3.0.0'
 [dependencies]
 jsonrpc-core = '15.1.0'
 structopt = '0.3.8'
-rand = "0.8.3"
+rand = "0.7.2"
```
Comment (stechu) [inline runtimes/manta/Cargo.toml:23]: please revert the downgrade?

**Comment by stechu** [comment]
@Dengjianping  I am working on this branch right now.

**Comment by stechu** [comment]
@Dengjianping @zhenfeizhang  I think this PR is ready to merge. Please take a look. 
Both 1 node mode and 5 node mode works on my local machine, despite the setup is quite tedious. Will be working on automate the deployment later. Some screen shots:

5node-manta
manta-5nodes-more

**Comment by Dengjianping** [comment]
LGTM. But there's just one place need to align in Readme.
This line needs to be modified due to the name of chain spec is changed:
```shell
docker run mantanetwork/manta:latest --chain manta --name "ILoveManta"
```
To
```shell
docker run mantanetwork/manta:latest --chain manta-testnet --name "ILoveManta"
```

**Comment by stechu** [comment]
@Dengjianping  fixed. thanks for the review.

**Review by stechu — APPROVED** [review]

**Pull request merged**